content/develop/connect/insight/copilot-faq.md

```diff
@@ -0,0 +1,54 @@
+---
+categories:
+- docs
+- operate
+- redisinsight
+linkTitle: Redis Copilot FAQ
+title: Redis Copilot FAQ
+weight: 1
+---
+
+## General questions
+
+### What is Redis Copilot?
+Redis Copilot is an AI-powered developer assistant that helps you learn about Redis, explore your Redis data, and build search queries in a conversational manner.  It is available in our flagship visual GUI developer tool, Redis Insight, as well as within the Redis public documentation (general chatbot). 
+
+### How does Redis Copilot work?
+Redis Copilot is powered by the [OpenAI API](https://platform.openai.com/docs/overview) platform. When it needs to provide context-aware assistance, such as within the **my data** chat in Redis Insight, Redis Copilot will use data from your connected database. Some of that data, such as indexing schemas and sample keys, may be transmitted to the Redis Copilot backend and OpenAI for processing.
+
+### What kind of tasks can Redis Copilot perform?
+
+Currently, Redis Copilot provides two primary features: a general chatbot and a context-aware data chatbot.
+
+**General chatbot**: the knowledge-based chatbot serves as an interactive and dynamic documentation interface where developers can ask specific questions about the Redis commands, concepts, and products, and get responses on the fly, simplifying the learning process. The general chatbot is also available in our public docs.
+
+**My data chatbot**: the context-aware chatbot enables users to construct search queries using everyday language rather than requiring specific programming syntax. This feature allows for intuitive query building and data exploring making it easier for developers to interact with their data without extensive technical knowledge.
+
+### How do I get started with Redis Copilot in Redis Insight?
+
+The Redis Copilot instance within our public documentation is free for anyone to use and is available now to answer general questions about Redis.
+
+The Redis Copilot instance that is embedded in Redis Insight is gradually being rolled out to its entire user base. Once you are granted access to Redis Copilot in the app, you need to sign in with your Redis Cloud account before you can start using it. If you don’t have an account with Redis Cloud yet, it will be automatically created upon sign in at no initial cost to you. 
```

Comment:
I'm not sure if "initial" is actually correct here, but it implies you might have to pay eventually :-)
```suggestion
The Redis Copilot instance that is embedded in Redis Insight is gradually being rolled out to the user base. Once you are granted access to Redis Copilot in the app, you need to sign in with your Redis Cloud account before you can start using it. If you don’t have an account with Redis Cloud yet, it will be automatically created when you sign in at no extra cost to you. 
```